draft-kazuho-httpbis-priority.md

```diff
@@ -260,6 +264,22 @@ TBD
 
 # IANA Considerations
 
-TBD
+This specification registers the following entry in the Permanent Message Header
+Field Names registry established by {{?RFC3864}}:
+
+* Header field name: Priority
+* Applicable protocol: http
+* Status: standard
+* Author/change controller: IETF
+* Specification document(s): [this document]
+* Related information:
+
+This specification registers the following entry in the HTTP/2 Settings registry
+established by {{?RFC7540}}:
```

Comment:
informative reference will conflict with normative?

Reply:
At least my build passes, and RFC7540 appears as normative in the generated I-D. My assumption that normative is given preference when there is a conflict.